core/src/main/java/org/elasticsearch/index/shard/IndexShardRelocatedException.java

```diff
@@ -29,10 +29,14 @@
 public class IndexShardRelocatedException extends IllegalIndexShardStateException {
```

Comment:
maybe while we are at it can we remove this exception entirely and just use `new IllegalIndexShardStateException(shardId, IndexShardState.RELOCATED, "Already relocated")` where it's used I think there is only one place really?